src/components/JaegerToolbar/TagsControl.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Col, Form, Card, FormGroup, FormControl, FieldLevelHelp } from 'patternfly-react';
+import { KialiAppState } from '../../store/Store';
+
+interface TagsControlProps {
+  fetching: boolean;
+  tags: string;
+  onChange: (event: any) => void;
+}
+
+export class TagsControl extends React.PureComponent<TagsControlProps, {}> {
+  constructor(props: TagsControlProps) {
+    super(props);
+  }
+
+  tagsHelp = () => {
+    return (
+      <Card>
+        <Card.Title>
+          Values should be in the{' '}
+          <a href="https://brandur.org/logfmt" target="_blank">
```

Comment:
we should add ` rel="noopener noreferrer"` to any link leading to third-parties, it prevents potential undesired data leak . See https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
ok, I didn't know. thanks

Reply:
same here, why isn't it done?